src/eko/harmonics/log_functions.py

```diff
@@ -2,6 +2,7 @@
 """Implementation of some Mellin transformation of functions as:
 
     :math:`(1-x)\\ln^k(1-x), \\quad k = 1,2,3`
+    :math:`\\ln^k(1-x), \\quad k = 1,3,4,5`
```

Comment:
is there a deeper reason why we are missing k=2?

Reply:
No, it's just not needed anywhere fr the time being.